index.html

```diff
@@ -1,25 +1,38 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Page Title</title>
+    <title>Girma's Page</title>
+    <meta charset="UTF-8" />
+    <meta name="keywords" content="HTML, CSS, JavaScript" />
   </head>
   <body>
-    <h1>Girma Ebssa</h1>
-    <h1>GG</h1>
-    <h2>About</h2>
-    <p>Ebssa G who is a CodeTheDream student attending io-programing.</p>
-    <h2>Experience</h2>
-    <ul>
-      <li>Ethiopia</li>
-      <li>Seattle</li>
-      <li>Atlanta</li>
-      <li>Atlanta</li>
-    </ul>
-    <h2>Connect</h2>
-    <ul>
-      <li><a href="https://www.linkedin.com/">Linkedin</a></li>
-      <li><a href="https://twitter.com/?lang=en">X</a></li>
-      <li><a href="https://github.com/?lang=en">GitHub</a></li>
-    </ul>
-  </body>
+    <nav>
+      <ul>
+        <li><a href="#About">About</a></li>
+        <li><a href="#Experience">Experience</a></li>
+        <li><a href="#Connect">Connect</a></li>
+      </ul>
+    </nav>
+    <section id="About">
+      <h1>Girma Ebssa</h1>
+      <h2>About</h2>
+      <p>Ebssa G who is a CodeTheDream student attending io-programing.</p>
+    </section>
+    <section id="Experience">
+      <h2>Experience</h2>
+      <ul>
+        <li>Ethiopia</li>
+        <li>Seattle</li>
+        <li>Atlanta</li>
+        <li>Atlanta</li>
+      </ul>
+    </section>
+    <section id="Connect">
```

Comment:
Good job adding all the section ids. This one doesn't look like it's closed though. You'll just want to add the `</section>` tag below the link list.